content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Mobile Session Replay Setup and Configuration
+kind: documentation
+description: Setting up and configuring Mobile Session Replay.
+aliases:
+further_reading:
+    - link: '/real_user_monitoring/session_replay/mobile'
+      tag: Documentation
+      text: Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay/mobile/app_performance'
+      tag: Documentation
+      text: How Mobile Session Replay Impacts App Performance
+    - link: '/real_user_monitoring/session_replay/mobile/privacy_options'
+      tag: Documentation
+      text: Mobile Session Replay Privacy Options
+    - link: '/real_user_monitoring/session_replay/mobile/troubleshooting'
+      tag: Documentation
+      text: Troubleshoot Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay'
+      tag: Documentation
+      text: Session Replay
+---
+
+## Setup
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+All Session Replay SDK versions can be found in the [Maven Snapshots Repository][1].
+
+To set up Mobile Session Replay for Android:
+
+1. Make sure you've [setup and initialized the Datadog Android RUM SDK][2] with views instrumentation enabled.
+
+2. Import Datadog Session Replay dependency.
```

Comment:
@mariusc83 Just wanted to check because it's not clear to me what this is. Is this a single file? Is it obvious what it is and how to find it after initializing the Android RUM SDK?

```suggestion
2. Import any Datadog Session Replay dependencies.
```

Reply:
Or is this what you meant?

```suggestion
2. Declare Datadog Session Replay as a [dependency](https://docs.datadoghq.com/real_user_monitoring/android/?tab=kotlin#declare-the-sdk-as-a-dependency).
```